libs/dotnet-sdk/Featureboard.DotnetSdk.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="7.0.0" />
```

Comment:
Do the references need to be dropped back too?

Reply:
No they seem to support both
![image](https://github.com/arkahna/featureboard-sdks/assets/84910463/c8263373-41e8-48cb-8b90-2faf88fa5367)